src/embed/hotspot-renderer.js

```diff
@@ -353,6 +353,12 @@ HotspotRenderer.prototype.focus_ = function(id) {
   this.tween = new TWEEN.Tween(hotspot.scale).to(FOCUS_SCALE, FOCUS_DURATION)
       .easing(TWEEN.Easing.Quadratic.InOut)
       .start();
+
+  if (this.worldRenderer.isVRMode()) {
+    this.timeForHospotClick = setTimeout( function () {
```

Comment:
Please don't use .bind! Use new ES6 syntax for declaring an anonymous function + closure like this:

setTimeout(() => {